pkg/controller.go

```diff
@@ -151,6 +156,9 @@ func createControllerContainers(csi *csibaremetalv1.Deployment) []corev1.Contain
 				{Name: "NAMESPACE", ValueFrom: &corev1.EnvVarSource{
 					FieldRef: &corev1.ObjectFieldSelector{APIVersion: "v1", FieldPath: "metadata.namespace"},
 				}},
+				{Name: fastTimeoutEnv, Value: c.FastTimeout},
+				{Name: slowTimeoutEnv, Value: c.SlowTimeout},
+				{Name: attemptsNumEnv, Value: strconv.Itoa(int(c.MaxAttempts))},
```

Comment:
Double conversion? `string -> int -> string` ?

Reply:
No, `MaxAttempts` is `uint`

Reply:
Why not `strconv.FormatUint` ?

Reply:
Fixed